server/publications.js

```diff
@@ -5,6 +5,10 @@ Meteor.publish(null, function () {
     return [
       Users.find({
         _id: this.userId
+      }, {
```

Comment:
Hm. It wasn't sending all fields for me before I added that. Seems like it included email and a couple of others by default, but I needed to use fields to get additional fields. Am I missing something there?